torch/nn/modules/activation.py

```diff
@@ -886,19 +886,13 @@ def extra_repr(self) -> str:
 def _check_arg_device(x: Optional[torch.Tensor]) -> bool:
     if x is None:
         return True
-    else:
-        return x.device.type in ["cpu", "cuda", torch.utils.backend_registration._privateuse1_backend_name]
-
-    return False
+    return x.device.type in ["cpu", "cuda", torch.utils.backend_registration._privateuse1_backend_name]
```

Comment:
Actually, it seems like this was done on purpose see 
https://github.com/pytorch/pytorch/pull/95653#discussion_r1119627765

Reply:
Cancelling the merge since this was done purposefully

Reply:
I reverted python code fixes and left only typos fixes